### ISSUE TITLE
Trigger events within react-spreadsheet

### ISSUE THREAD

**Issue by benediktvaldez** [iddan/react-spreadsheet#128]
Is there a way to trigger events like “exit edit mode”? This would specially be useful for dropdowns (`react-select`'s `onMenuClose` for example) since closing it is the UX equivalent of “I'm done editing this for now”

**Comment by iddan** [comment]
This is definitely possible. I'm pretty busy right now but PRs are welcome

**Comment by iddan** [comment]
Can you specify which events would you like the component to expose?

**Comment by phen0menon** [comment]
Hello. I need to stop editing mode after calling `onChange` in the custom DataEditor component:

```js
const SelectEdit = ({ cell, options, onChange }) => {
  const handleChange = React.useCallback(
    (value: any) => {
      onChange({ ...cell, value: value ? value.value : null });
      // onChange is triggered, how to exit edit mode now?
    },
    [cell, onChange]
  );

  const option = React.useMemo(() => cell && options.find(o => o.value === cell.value), [
    cell,
    options
  ]);

  return (
    <CustomSelect value={option} onChange={handleChange} options={options} isEdit />
  );
};
```

How can I achieve that behavior?

**Comment by iddan** [comment]
This can not be achieved in the current API. It can be added to the DataEditor API I’ll try to get around it in the next release

**Comment by phen0menon** [comment]
@iddan I'm looking forward to it, a very important feature! When the next release will be, approximately?

**Comment by iddan** [comment]
I believe in the following days

**Comment by iddan** [comment]
Fixed in `react-spreadsheet@0.5.16`

**Comment by iddan** [comment]
https://iddan.github.io/react-spreadsheet/docs/api#dataeditorprops
